perllib/FixMyStreet/DB/ResultSet/Contact.pm

```diff
@@ -7,6 +7,17 @@ use POSIX qw(strcoll);
 
 __PACKAGE__->load_components('Helper::ResultSet::Me');
 
+
+sub fake {
```

Comment:
A comment explaining why this code is needed would be helpful.

Reply:
At the end of what I have done, I am not sure it is worth it, it only saved what, a line or two of code (and then in Part 2 I removed one of the two places it's used!). I guess it is slightly more consistent but not entirely so, e.g. admin/templates still calls stash_category_groups with a hash rather than an object.

Reply:
Have ditched it